### DISCUSSION TITLE
simultaneous delivery to email and SF case

### DISCUSSION THREAD

**Discussion by vf-kenh** [amazon-connect/amazon-connect-salesforce-scv#106] [Q&A]
I can't seem to find documentation indicating we can deliver simultaneously of the chosen methods. For example delivery of email AND SF case for the same contact.

Is this possible, or we have to opt for one method only? 
If so, how to?

**Comment by dougjaso** [comment]
Simultaneous delivery is not a part of the default code, however you could modify the functions to do this. You would need to modify the IF statement starting at 191. My first approach would be to set mutliple modes in the vm_mode contact attribute, then change the IF statement to look for substrings in that attribute. But you can take whatever approach you like.

**Comment by dcarbonetti** [comment]
You could just create the case, and have automation in salesforce to fire off the email
